frontend/src/api/elections.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export const createElection = async electionData => {
+  const { data } = await axios.post("/election/create-election", electionData);
+  return data;
+};
+
+export const getActiveElections = async () => {
+  const { data } = await axios.get("/election/active-elections");
+  return data.elections;
+};
+
+export const getInactiveElections = async () => {
+  const { data } = await axios.get("/election/inactive-elections");
+  return data.elections;
```

Comment:
this is not good. The api interfaces should always be consistent. please return the axios result directly. Having it like it is now is bad because different API functions return different object. Please update all API functions, and places where they are used.